apps/discovery_api/test/integration/discovery_api/auth/auth_test.exs

```diff
@@ -65,7 +66,7 @@ defmodule DiscoveryApi.Auth.AuthTest do
       auth0_setup()
       |> on_exit()
 
-      user = Helper.create_persisted_user(AuthHelper.valid_jwt_sub())
+      {user, _} = AuthHelper.login()
```

Comment:
Not sure I love this helper, as it's a bit ambiguous what token belongs to it. I may just have it return the token it used. Eh